Objects/Objects/BuiltElements/Topography.cs

```diff
@@ -21,23 +21,16 @@ public class Topography : Base, IDisplayMesh, IDisplayValue<List<Mesh>>
 
     public Topography()
     {
-      this.displayMesh = new Mesh();
+      displayValue = new List<Mesh>();
     }
 
     [SchemaInfo("Topography", "Creates a Speckle topography", "BIM", "Architecture")]
     public Topography([SchemaMainParam] Mesh displayMesh)
     {
-      this.displayMesh = displayMesh;
+      displayValue = new List<Mesh>{displayMesh};
```

Comment:
And here too, but this works just like is used to, now using displayValue directly instead of letting the `displayMesh` set the list.